### ISSUE TITLE
Transfer learning model only predict same class, even though has 96-99% val_acc

### ISSUE THREAD

**Issue by PreVizsla** [keras-team/keras#14185]
I tried to make a transfer learning model using Xception & InceptionV3 and other models
to classify the rock paper scissors dataset. The data set are balanced with 700 images on each category and same resolution of 200*200. But the problem is that this model only predict 1 class when I use it to predict it's own training dataset.

I tried to use other models such as InceptionV3 and still same problems occurs.
I tried to use the EXACT identic code with my own cnn architecture and surprisingly it works and can predict 3 class with great accuracy!! 
But I still couldn't find out what's wrong with my transfer learning method.
Can anyone help me to implement transfer learning correctly?

`from tensorflow.keras.preprocessing.image import ImageDataGenerator

IMAGE_SIZE = (200, 200)
BATCH_SIZE = 16
SEED = 72

def get_set():
    train_path = "/tmp/data/rockpaperscissors/rps-cv-images"

    train_gen = ImageDataGenerator(rescale=1./255, validation_split=0.4, 
                                   rotation_range=20,
                                    width_shift_range=0.2,
                                    height_shift_range=0.2,
                                    shear_range=0.2,
                                    zoom_range=0.2)
    train_set = train_gen.flow_from_directory(train_path, target_size=IMAGE_SIZE, \
                                              batch_size=BATCH_SIZE,seed=SEED, \
                                              subset='training')
    val_set = train_gen.flow_from_directory(train_path, target_size=IMAGE_SIZE, \
                                            batch_size=BATCH_SIZE,seed=SEED, \
                                            subset='validation')
    return train_set, val_set

train_set,val_set = get_set()`
(THIS BELOW PART IS THE ONLY DIFFERENCE FROM MY OWN CNN PROGRAM, BUT THE REST IS THE SAME, compile and fitting are the same)
`from tensorflow.keras.applications import Xception
from tensorflow.keras.models import Sequential
from tensorflow.keras.layers import Dense, Dropout, Activation, Flatten
from tensorflow.keras.layers import Conv2D, MaxPooling2D,  BatchNormalization

IMAGE_SIZE = (200, 200)
base = Xception(input_shape=(200,200,3),
                    include_top=False,      
                    pooling='avg',
                     weights='imagenet')
base.trainable = False
model = Sequential()
model.add(base)
model.add(Dense(512, activation='relu'))
model.add(Dense(3, activation='softmax'))
model.compile(loss='categorical_crossentropy',
                optimizer='adam',
                metrics=['accuracy'])
EPOCHS = 10
history = model.fit(train_set, epochs=EPOCHS, steps_per_epoch=25,
      validation_data=val_set,
      validation_steps=5,
      verbose=2)
`
Log:
Epoch 1/10
25/25 - 8s - loss: 0.6540 - accuracy: 0.7409 - val_loss: 0.2163 - val_accuracy: 0.9500
Epoch 2/10
25/25 - 6s - loss: 0.2179 - accuracy: 0.9326 - val_loss: 0.1635 - val_accuracy: 0.9375
Epoch 3/10
25/25 - 6s - loss: 0.1221 - accuracy: 0.9611 - val_loss: 0.2332 - val_accuracy: 0.9125
Epoch 4/10
25/25 - 6s - loss: 0.0918 - accuracy: 0.9725 - val_loss: 0.1365 - val_accuracy: 0.9375
Epoch 5/10
25/25 - 6s - loss: 0.0816 - accuracy: 0.9725 - val_loss: 0.1073 - val_accuracy: 0.9625
Epoch 6/10
25/25 - 6s - loss: 0.1327 - accuracy: 0.9534 - val_loss: 0.1814 - val_accuracy: 0.9375
Epoch 7/10
25/25 - 6s - loss: 0.1308 - accuracy: 0.9600 - val_loss: 0.0794 - val_accuracy: 0.9750
Epoch 8/10
25/25 - 6s - loss: 0.0751 - accuracy: 0.9725 - val_loss: 0.2213 - val_accuracy: 0.9375
Epoch 9/10
25/25 - 7s - loss: 0.0938 - accuracy: 0.9585 - val_loss: 0.1409 - val_accuracy: 0.9500
Epoch 10/10
25/25 - 6s - loss: 0.0845 - accuracy: 0.9675 - val_loss: 0.1113 - val_accuracy: 0.9750

**Comment by jvishnuvardhan** [comment]
@PreVizsla Sorry for the late response. Is this still an issue for you?

I ran your code and the predictions are not same as you reported. Can you please check the [gist here](https://colab.research.google.com/gist/jvishnuvardhan/c3a702b28b4ce3c653fe65f1a2d14dec/untitled.ipynb). Thanks!
